sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveExternalCatalogSuite.scala

```diff
@@ -154,4 +155,31 @@ class HiveExternalCatalogSuite extends ExternalCatalogSuite {
     catalog.createTable(hiveTable, ignoreIfExists = false)
     assert(catalog.getTable("db1", "spark_29498").owner === owner)
   }
+
+  test("SPARK-30868 throw an exception if HiveClient#runSqlHive fails") {
+    // test add jars which not exists
+    val jarPath = "file:///tmp/not_exists.jar"
+    assertThrows[QueryExecutionException](externalCatalog.client.runSqlHive(s"ADD JAR $jarPath"))
+
+    // test change to the database which not exists
+    assertThrows[QueryExecutionException](externalCatalog.client.runSqlHive(
+      s"use db_not_exists"))
+
+    // test create hive table failed
+    val tblName = "table_not_exists"
+    assertThrows[QueryExecutionException](externalCatalog.client.runSqlHive(
+      s"CREATE TABLE $tblName(n into)"))
+
+    // test describe table failed
+    assertThrows[QueryExecutionException](externalCatalog.client.runSqlHive(
+      s"DESC FORMATED $tblName"))
```

Comment:
I'd keep either `DESC` or `DROP`, to test non-existing table.

Reply:
This is an abnormal DESC statement, the `FORMATED` is a wrong word.

Reply:
ah i see, then the table name doesn't matter. how about `DESC FORMATED t`? and add comment to say that `FORMATED` is a wrong word.